config/develop/nextflow-vpc-endpoint.yaml

```diff
@@ -0,0 +1,10 @@
+template_path: remote/gateway-vpc-endpoint.yaml
+stack_name: nextflow-vpc-endpoint
```

Comment:
i would probably add `s3` this stack name because you may want to add other endpoints between this vpc and another service.  how about something like `s3-nextflow-vpc-endpoint`?

Reply:
Good idea!